tests/Constraints/AdditionalPropertiesTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * This file is part of the Gruvo software.
```

Comment:
I removed it. It was added automatically by my IDE and I just overlooked it. Sorry, @yuloh .